sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentAnalysisClientOptions.cs

```diff
@@ -28,9 +28,6 @@ public DocumentAnalysisClientOptions(ServiceVersion version = LatestVersion)
             };
 
             AddLoggedHeadersAndQueryParameters();
-
-            //Default Audience to Azure Public Cloud
-            Audience ??= DocumentAnalysisAudience.AzurePublicCloud;
```

Comment:
Fixes #29019. We agreed on defaulting this property to null. Public Cloud will be assumed by the client if left `null`.

Reply:
Is this something that came in archboard review - I ask because I want to know if this is something we want to do across all SDKs

Reply:
Yeah I remember that conversation but I was not sure if there was any conversation across the board :)